### PR TITLE
chore: normalize characteristic points releated properties

### PR DESCRIPTION
**Pull request by ArdtK**
## Issue addressed
Solves #207 (first step)

## Code of conduct
- [x] I HAVE NOT added sensitive or compromised (test) data to the repository.
- [x] I HAVE NOT added vulnerabilities to the repository.
- [ ] I HAVE discussed my solution with (other) members of the KOSWAT team.

## What has been done?
- Normalized to the agreements stated in the issue.
- Translated all variable names in the foms to English.

### Checklist
- [x] Tests are either added or updated.
- [x] Branch is up to date with `master`.
- [x] Updated documentation if needed.

## Additional Notes (optional)
Add any additional notes or information that may be helpful.

### CODE REVIEW

File: koswat/configuration/io/csv/koswat_traject_surroundings_csv_fom.py
Comment (Carsopre) [inline koswat/configuration/io/csv/koswat_traject_surroundings_csv_fom.py:1]: Should the file be renamed? the class has also a different one.

File: koswat/configuration/io/ini/koswat_costs_ini_fom.py
Comment (Carsopre) [inline koswat/configuration/io/ini/koswat_costs_ini_fom.py:1]: Great addition for non-dutch speakers thanks!

**Review by Carsopre — APPROVED** [review]
Minor remark in regards of file renaming.

File: koswat/configuration/io/csv/koswat_traject_surroundings_csv_fom.py
Comment (ArdtK) [reply koswat/configuration/io/csv/koswat_traject_surroundings_csv_fom.py:1]: I reverted it back to `traject` as there will be another issue for the dike/section/traject stuff.

**Pull request merged**